curriculum/challenges/english/05-apis-and-microservices/basic-node-and-express/meet-the-node-console.english.md

```diff
@@ -12,7 +12,11 @@ We recommend to keep the log panel open while working at these challenges. By re
 
 ## Instructions
 <section id='instructions'>
-Modify the <code>myApp.js</code> file to log "Hello World" to the console. 
+  
```

Comment:
There are two spaces on this line instead of an empty line, which causes the markdown to not render properly.
```suggestion

```

Reply:
Was it rendering as code?  I'm glad you caught that - not sure how I missed it.

Reply:
It was rendering as code.

Reply:
Thanks for the pickup!